packages/knip/fixtures/plugins/msw/msw.toml

```diff
@@ -0,0 +1 @@
+[[plugins]]
```

Comment:
Does MSW allow for configuration files? If so, and if it could contain references to dependencies, the file patterns could added to `CONFIG_FILE_PATTERNS`. Otherwise this file does not serve any purpose and can be removed.

Reply:
> Does MSW allow for configuration files?

Not that I know of, I'll remove it.